lib/datadog/statsd.rb

```diff
@@ -151,6 +151,7 @@ def self.open(*args)
     # @param [String] stat stat name
     # @param [Hash] opts the options to create the metric with
     # @option opts [Numeric] :sample_rate sample rate, 1 for always
+    # @option opts [Boolean] :pre_sampled if true, do not sample the data but still reports the sample rate passed in
```

Comment:
This mixes imperative form ("do ..") and present tense ("reports").  How about
```
If true, the client assumes the caller has already sampled metrics at :sample_rate, and does perform sampling.
```

Reply:
This version sounds better than mine!